### PR TITLE
Less and beter errors with check_es on OpenShift

### PR DESCRIPTION
**Pull request by sbrunner**
Fix GSGMF-1598

### CODE REVIEW

File: c2cwsgiutils/scripts/check_es.py [hunk 1]
```diff
@@ -66,10 +68,19 @@ def _check_roundtrip() -> None:
     query = {"query": {"match_phrase": {"log.logger": logger_name}}}
     start = time.monotonic()
     while time.monotonic() < start + LOG_TIMEOUT:
-        for _ in range(10):
-            r = requests.post(SEARCH_URL, json=query, headers=SEARCH_HEADERS)
+        exception = None
+        for _ in range(int(os.environ.get("C2CWSGIUTILS_CHECKES_TRYNUMBER", 10))):
```
Comment (ger-benjamin) [inline c2cwsgiutils/scripts/check_es.py:72]: One check, multiple checks. not checkes. So:
=> 
```suggestion
        for _ in range(int(os.environ.get("C2CWSGIUTILS_CHECKS_TRYNUMBER", 10))):
```

or C2CWSGIUTILS_CHECK_ES_TRYNUMBER ?

File: c2cwsgiutils/scripts/check_es.py [hunk 2]
```diff
@@ -66,10 +68,19 @@ def _check_roundtrip() -> None:
...
+            try:
+                r = requests.post(SEARCH_URL, json=query, headers=SEARCH_HEADERS)
+                exception = None
+            except HTTPConnection as e:
+                logger.exception("Error on querring Elasticsearch")
+                exception = e
             if r.ok:
                 continue
+            time.sleep(float(os.environ.get("C2CWSGIUTILS_CHECKES_SLEEP", 1)))
```
Comment (ger-benjamin) [inline c2cwsgiutils/scripts/check_es.py:81]:
```suggestion
            time.sleep(float(os.environ.get("C2CWSGIUTILS_CHECKS_SLEEP", 1)))
```

Or C2CWSGIUTILS_CHECK_ES_SLEEP ?

File: c2cwsgiutils/scripts/check_es.py [hunk 3]
```diff
@@ -92,17 +103,21 @@ def deprecated() -> None:
...
+            max_ts = _max_timestamp()
+        now = datetime.datetime.now(max_ts.tzinfo)
+        age = round((now - max_ts).total_seconds())
+        LOG.info("Last log age: %ss", age)
+        stats.set_gauge(["max_age"], age)
+
+        if "LOG_TIMEOUT" in os.environ:
+            _check_roundtrip()
+    except:  # pylint: disable=bare-except
+        LOG.exception("Exception durring run")
```
Comment (ger-benjamin) [inline c2cwsgiutils/scripts/check_es.py:119]:
```suggestion
        LOG.exception("Exception during run")
```

**Review by ger-benjamin — APPROVED** [review]
Looks good !

File: c2cwsgiutils/scripts/check_es.py [hunk 4]
```diff
@@ -66,10 +68,19 @@ def _check_roundtrip() -> None:
     query = {"query": {"match_phrase": {"log.logger": logger_name}}}
     start = time.monotonic()
     while time.monotonic() < start + LOG_TIMEOUT:
-        for _ in range(10):
-            r = requests.post(SEARCH_URL, json=query, headers=SEARCH_HEADERS)
+        exception = None
+        for _ in range(int(os.environ.get("C2CWSGIUTILS_CHECKS_TRYNUMBER", 10))):
```
Comment (arnaud-morvan) [inline c2cwsgiutils/scripts/check_es.py:72]:
```suggestion
        for _ in range(int(os.environ.get("C2CWSGIUTILS_CHECK_ES_RETRIES", 10))):
```

File: c2cwsgiutils/scripts/check_es.py [hunk 5]
```diff
@@ -66,10 +68,19 @@ def _check_roundtrip() -> None:
...
+            try:
+                r = requests.post(SEARCH_URL, json=query, headers=SEARCH_HEADERS)
+                exception = None
+            except HTTPConnection as e:
+                logger.exception("Error on querring Elasticsearch")
```
Comment (arnaud-morvan) [inline c2cwsgiutils/scripts/check_es.py:77]:
```suggestion
                logger.exception("Error on querying Elasticsearch")
```

File: c2cwsgiutils/scripts/check_es.py [hunk 6]
```diff
@@ -66,10 +68,19 @@ def _check_roundtrip() -> None:
...
+                exception = e
             if r.ok:
                 continue
+            time.sleep(float(os.environ.get("C2CWSGIUTILS_CHECKS_SLEEP", 1)))
```
Comment (arnaud-morvan) [inline c2cwsgiutils/scripts/check_es.py:81]:
```suggestion
            time.sleep(float(os.environ.get("C2CWSGIUTILS_CHECK_ES_SLEEP", 1)))
```

**Review by arnaud-morvan — APPROVED** [review]

**Pull request merged**